packages/amplify_auth_plugin_interface/lib/src/SignUp/AuthNextSignUpStep.dart

```diff
@@ -18,5 +18,5 @@ import '../types/AuthNextStep.dart';
 
 class AuthNextSignUpStep extends AuthNextStep {
   String signUpStep;
-  AuthNextSignUpStep({additionalInfo, @required codeDeliveryDetails, @required this.signUpStep}) : super(additionalInfo: additionalInfo, codeDeliveryDetails: codeDeliveryDetails);
+  AuthNextSignUpStep({required additionalInfo, required codeDeliveryDetails, required this.signUpStep}) : super(additionalInfo: additionalInfo, codeDeliveryDetails: codeDeliveryDetails);
```

Comment:
Necessary to make `additionalInfo` required?

Reply:
You're right it's not required as AuthNextStep provides a default value. Made it optional.